app/src/main/java/org/mozilla/fenix/historymetadata/controller/HistoryMetadataController.kt

```diff
@@ -27,13 +33,20 @@ interface HistoryMetadataController {
      * @see [HistoryMetadataInteractor.onHistoryMetadataGroupClicked]
      */
     fun handleHistoryMetadataGroupClicked(historyMetadataGroup: HistoryMetadataGroup)
+
+    /**
+     * @see [HistoryMetadataInteractor.onRemoveItems]
+     */
+    fun handleRemoveItems(keys: List<HistoryMetadataKey>)
```

Comment:
I think with the new designs, this can just be `handleRemoveGroup` as we don't display individual items anymore.

Reply:
awesome! was gonna check with Nicole if we were planning on adding individual items back in